### PR TITLE
IGAPP-3: CI

### PR DESCRIPTION
**Pull request by klinzo**

### CODE REVIEW

File: .circleci/src/workflows/scheduled_production_delivery.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        - web_build
+  - await_approval:
+      type: approval
+      requires:
+        - web_prepare_delivery
+  - web_deliver:
+      production_delivery: true
+      requires:
+        - await_approval
+
```
Comment (Taggotty) [inline .circleci/src/workflows/scheduled_production_delivery.yml:22]: pls remove newline

File: native/package.json [hunk 2]
```diff
@@ -29,8 +29,7 @@
     "flow:install-types": "flow-typed install --ignoreDeps dev",
     "flow:install-types:all": "flow-typed install",
     "postinstall": "jetifier",
-    "locales:missing": "node locales/tools/findMissingLocales",
-    "circleci:update-config": "cd .circleci && { cat autogenerated_header.yml; circleci config pack src; } > config.yml"
+    "locales:missing": "node locales/tools/findMissingLocales"
```
Comment (Taggotty) [inline native/package.json:32]: can be removed, as max will move the locales to the root

**Review by Taggotty — APPROVED** [review]

File: native/package.json (hunk 2 above)
Comment (klinzo) [reply native/package.json:32]: Then max will do that :D

**Review by maxammann — APPROVED** [review]

File: native/package.json (hunk 2 above)
Comment (maxammann) [inline native/package.json:32]:
```suggestion
```

File: native/package.json [hunk 3]
```diff
@@ -29,8 +29,7 @@
     "flow:install-types": "flow-typed install --ignoreDeps dev",
     "flow:install-types:all": "flow-typed install",
     "postinstall": "jetifier",
```
Comment (maxammann) [inline native/package.json:31]:
```suggestion
    "postinstall": "jetifier"
```

**Review by maxammann — APPROVED** [review]

**Pull request merged**